itchef/cookbooks/cpe_chrome/libraries/gen_windows_chrome_known_settings.rb

```diff
@@ -310,6 +310,12 @@ module KnownSettings
           :dword,
           false,
         ),
+        'CloudReportingEnabled' => WindowsChromeFlatSetting.new(
```

Comment:
Seems reasonable, but could you also add the relevant registry info to [this file](https://github.com/facebook/IT-CPE/blob/e92a1f3cc97672df0196de35105b059e64dc1405/itchef/cookbooks/cpe_chrome/scripts/win_chrome_manual_policy.reg#L5) so this doesn't get overridden in the future?

Reply:
All set